ansible_base/urls.py

```diff
@@ -1,42 +1,17 @@
 import logging
```

Comment:
Don't provide a urls.py for ansible_base. We should be explicitly declaring routes for the various apps here in each service.

Reply:
Each app has a URL and we provide a `lib/dynamic_config/dynamic_urls.py` as a convenience method to auto-load urls from an including app.